templates/gunicorn_start.j2

```diff
@@ -11,7 +11,7 @@ exec gunicorn \
     --workers {{ crosshair_gunicorn_workers }} \
     --timeout {{ crosshair_gunicorn_timeout }} \
     --user gunicorn \
-    --group crosshair \
+    --group gunicorn \
```

Comment:
Getting permission issues here. No reason to use crosshair for this I don't think?